packages/dialtone-vue3/components/popover/popover.vue

```diff
@@ -953,6 +968,48 @@ export default {
         onShow: this.onShow,
       });
     },
+
+    //  ============================================================================
+    //  $ HOVERCARD
+    //  ----------------------------------------------------------------------------
+
+    setInTimer () {
+      this.inTimer = setTimeout(() => {
+        this.isOpen = true;
+      }, TOOLTIP_DELAY_MS);
+    },
+
+    setOutTimer () {
+      this.outTimer = setTimeout(() => {
+        this.isOpen = false;
+      }, TOOLTIP_DELAY_MS);
+    },
+
+    onEnterAnchor (e) {
+      if (!this.hovercard) return;
+      clearTimeout(this.outTimer);
+      this.setInTimer(e);
+    },
+
+    onLeaveAnchor (e) {
+      if (!this.hovercard) return;
+      clearTimeout(this.inTimer);
+      this.setOutTimer(e);
+    },
+
+    onEnterContent () {
+      if (!this.hovercard) return;
+      clearTimeout(this.outTimer);
+    },
+
+    onLeaveContent (e) {
+      if (!this.hovercard) return;
+      this.setOutTimer(e);
+    },
```

Comment:
Does it make sense to do this timing logic here? Or does it make more sense for this popover to simply emit a "mouseover", "mouseleave" event and all the timing stuff is handled within the hovercard component?

If this were the case we could probably remove the "hovercard" prop from this component which doesn't quite feel right.

Reply:
Yes, it can be done on hovercard side if we use "mouseover", "mouseleave" events.

But,
Since we are reusing popover component to hovercard I would prefer to have the logic in there.
Then hovercard component does not have any logic, it is just a "popover-like variant", thats the idea to have the `hovercard` prop on popover.

Regarding emitting events, if you are not using hovercard on popover, you will emit useless events to the consumer.

Reply:
Yeah it's a bit of a tradeoff, fine to do it this way I think.